### PR TITLE
Menus to xml

### PR DESCRIPTION
**Pull request by NilsHullegien**
Converted our Server screens to .XML instead of the java code. 
I also added screens for android, but since they don't have a use and aren't connected to anything, nor have any functionality added, we can consider leaving those out for this PR.

### CODE REVIEW

**Comment by dVoorhout** [comment]
looks good

**Comment by dVoorhout** [comment]
(checks fail because of failing jmonkey servers)

File: ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java [hunk 1]
```diff
@@ -57,256 +50,12 @@ public void initMenu(AssetManager assetManager, InputManager inputManager
...
-                         panel(panelWithControl("panel_botton_left", "center", "none", "center", "#44f8", "50%", "50%",
-                                 createButtonControl("StartButton", "Start", "none", "center", "center", "50%", "50%", "toScreen(host)")));
-                         panel(panelWithControl("panel_botton_right", "center", "none", "center", "#88f8", "50%", "50%",
-                                 createButtonControl("QuitButton", "Quit", "none", "center", "center", "50%", "50%", "quitGame()")));
-
-                     }});
-             }});
-
-         }}.build(nifty));
+         nifty.fromXml("com/github/migi_1/Context/screens/main_screen.xml", "start", new MainMenuFunctions());
```
Comment (LMiljak) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java]: I think this should be moved to src/main/resources (default maven resource folder). This applies to all .xml files.

**Comment by NilsHullegien** [comment]
Closing branch for now for to merge with changed app states.

**Comment by NilsHullegien** [comment]
Resources are added in a different package in src/main/java, since this is the default location of nifty when it wants to import files. Also merged this branch with master so that it has the refactory of the appStates branch.

**Comment by NilsHullegien** [comment]
(Travis still fails because the jmonkey servers are down)

**Comment by LMiljak** [comment]
You didn't really need to change the package if JMonkey doesn't recognise the standard resource folder. This is still fine though. Merging.

**Pull request merged**